scripts/bert/data/preprocessing_utils.py

```diff
@@ -381,3 +386,119 @@ def convert_squad_examples(record, is_training):
         start_position=start_position, end_position=end_position, start_offset=answer_offset,
         end_offset=answer_offset + len(orig_answer_text) - 1, is_impossible=is_impossible)
     return example
+
```

Comment:
The functions added here seem XLNet specific. Let's not add them to `scripts/bert` then?

Reply:
Hi, it includes both BERT & XLNet features. Actually for XLNet, they are all raw feautres like start/end offset and for BERT, there are some processed features like start/end position.
Maybe I can split them up and move to scripts/language_model together with the following XLNet specific functions?